RocketChat/Rocket.Chat
# fix: New livechat conversations are not assigned to contact manager

## Proposed changes (including videos or screenshots)

 - Consider the `contactManager` property stored in **contacts** when deciding the default new agent to assign to a livechat room (instead of only considering the `contactManager` field in visitors, which is not maintained anymore).

## Issue(s)


## Steps to test or reproduce

In a **Premium** workspace, enable the "Assign new conversations to the contact manager" setting under **Settings > Omnichannel > Routing**. Then create a new contact manually (in the livechat contact center) using a livechat agent and assign yourself user as the contact manager (use the `contacts.get` endpoint to make sure the `contactManager` field contains the user id of the agent who created the contact). Then start a new livechat conversation using the info from the contact that has just been created.
**Expected result:** the contact manager should be automatically assigned to the room if it is online.

**Note:** use "Auto Selection" as the "Omnichannel Routing Method" (under **Settings > Omnichannel > Routing**). Otherwise, only bot contact managers will be auto assigned to rooms.

## Further comments

[CORE-862]

[CORE-862]: https://rocketchat.atlassian.net/browse/CORE-862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
